packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -713,7 +713,7 @@ export class Viewer {
     this.initializeDefaultKeyboardInteractionHandlers();
   }
 
-  /**
+  /**re
```

Comment:
Is the 're' needed?

Reply:
Nope, guessing I meant to type this into the terminal and accidentally saved. Good catch! I'll remove